### ISSUE TITLE
Add requirements file

### ISSUE THREAD

**Issue by vigliensoni** [napulen/encoding_matters#1]
In order to facilitate the reproducibility of the experiments, a file with all the modules and libraries required would be helpful.

**Comment by napulen** [comment]
Agree. I will add that as the last step, and probably a release tag with whatever status the repository has at the moment of submission. Thanks for checking the repo.
